lib/controller/EetplanController.class.php

```diff
@@ -92,6 +97,27 @@ public function woonoorden($actie = null) {
         }
     }
 
+    public function bekendehuizen($actie = null) {
+        if ($actie == 'toevoegen') {
+            $woonoord = filter_input(INPUT_POST, 'woonoord');
+            $uid = filter_input(INPUT_POST, 'uid');
+
+            $eetplan = new Eetplan();
+            $eetplan->avond = 0;
+            $eetplan->uid = $uid;
+            $eetplan->woonoord_id = $woonoord;
+            $form = new EetplanBekendeHuizenForm($eetplan);

```

Comment:
Formulier pikt die input parameters vanzelf op als de fields dezelfde naam hebben als de properties:
```
	private function loadProperty(InputField $field) {
		$fieldName = $field->getName();
		if ($this->model instanceof PersistentEntity AND property_exists($this->model, $fieldName)) {
			$this->model->$fieldName = $field->getValue();
		}
	}
```
Je hoeft dus alleen een nieuw of bestaand Eetplan mee te geven en niet met de hand uid en woonoord_id te zetten.